public/utils/helpers.js

```diff
@@ -46,3 +47,17 @@ export const inputLimitText = (
     </EuiText>
   );
 };
+
+export async function getAlerts({ params, httpClient, notifications, location, history }) {
+  const queryParamsString = queryString.stringify(params);
+  history.replace({ ...location, search: queryParamsString });
+
+  const resp = await httpClient.get('../api/alerting/alerts', { query: params });
+
+  if (resp.ok) {
+    return { alerts: resp.alerts, totalAlerts: resp.totalAlerts };
+  } else {
+    console.log('Error getting findings:', resp);
+    backendErrorNotification(notifications, 'get', 'findings', resp.err);
+  }
```

Comment:
I think it may be a good idea to wrap this logic in a `try/catch` block, or chaining a [.catch()](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/public/pages/Dashboard/containers/Dashboard.js#L225) call after the `.then()`. The contents of that `catch` block could be something like:
```
console.log('Error getting alerts: ", error);
backendErrorNotification(notifications, 'get', 'findings', error);
```